src/Evolution/Particles/MonteCarlo/NeutrinoInteractionTable.hpp

```diff
@@ -25,6 +26,26 @@ class NeutrinoInteractionTable {
   /// Read table from disk and stores interaction rates.
   explicit NeutrinoInteractionTable(const std::string& filename);
 
+  /// Explicit instantiation from table values, for tests
+  explicit NeutrinoInteractionTable(
```

Comment:
Doesn't need `explicit` since it takes multiple arguments.

Can you move the definition to the cpp file? Constructors are secretly a lot to compile :)